app/grandchallenge/core/templates/site.html

```diff
@@ -28,8 +28,8 @@
 {% block jumbotron %}
     {% if site.banner %}
         <div id="headerTitle"
-             class="container">
-            <div class="row mt-3">
+             class="container mt-3">
+            <div class="row p-3">
```

Comment:
Check that the columns do not jump around when going between pages with banners, pages without banners, and full width pages (eg, the all challenges page)

Reply:
What do you mean with jump around?

Reply:
The top left corner of the banner and the full width card should be at the same position on the screen, so that when you're clicking through pages (without scrolling in between) it doesn't look like the cards are moving left/right/up/down.